Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/SecurityAndComplianceV2.yml

```
@@ -1149,9 +1149,42 @@ Msg: Export to .msg message files."
       predefined:
       - 'true'
       - 'false'
+  - arguments:
+    - description: Identity of the policy to update.
+      name: Identity
```

```suggestion
      name: identity
```
Argument names are in lower case